index.js

```diff
@@ -267,6 +267,40 @@ client.on("voiceStateUpdate", (oldState, newState) => {
             delete callRecords[userId];
         }
     }
+
+    // Check emojis for deleting todo:
+    const message = reaction.message.content;
+    const regex = /<@(\d+)>/g;
```

Comment:
Could you make this regex into a constant? And then move it up to the top of the file? If you keep this RegEx in this block, it's going to be recompiled every single time this block is run, which is very slow and vulnerable to [RegEx DoS](https://owasp.org/www-community/attacks/Regular_expression_Denial_of_Service_-_ReDoS).